release/preview/windowsservercore/getLatestTag.ps1

```diff
@@ -14,6 +14,6 @@ $modulePath = Join-Path -Path $repoRoot -ChildPath 'tools\getDockerTags'
 Import-Module $modulePath -Force
 
 # The versions of nanoserver we care about
-$shortTags = @('1809','1903','1909')
+$shortTags = @('1809', '1903', '1909', '2004')
```

Comment:
This should be moved into meta.json

see lts, but not a required change

Reply:
Easy, so I did it.